eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -44,10 +37,12 @@ func (c *ClusterLoader) Apply() (err error) {
 		*eksconfig.ClusterLoaderSpec
 		ConfigMapData map[string]string
 		TestArgs      string
+		InstanceTypes []string
 	}{
 		ClusterLoaderSpec: c.Config.Spec.ClusterLoader,
 		ConfigMapData:     configMapData,
 		TestArgs:          c.buildArgs(),
+		InstanceTypes:     c.Config.Spec.ClusterLoader.InstanceTypes,
```

Comment:
IMO instance types shouldn't be used to select. Perhaps we can use a node selector?